tests/test_views_search.py

```diff
@@ -65,6 +64,24 @@ def test_page_size(app, indexed_records, search_url):
         assert 'message' in get_json(res)
 
 
+@pytest.mark.parametrize('app', [dict(
+    records_rest_endpoints=dict(
+        recid=dict(
+            max_result_window=2,
+        )
+    ),
+)], indirect=['app'])
+def test_next_in_max_page(app, indexed_records, search_url):
+    """Test page and size parameters."""
+    with app.test_client() as client:
+        res = client.get(search_url, query_string=dict(page=1, size=2))
+        assert "next" in res.json['links']
+
+        res = client.get(search_url, query_string=dict(page=2, size=2))
```

Comment:
Can you test with url from next page link from previous response instead of query string?